extensions/amp-consent/0.1/consent-ui.js

```diff
@@ -36,9 +37,9 @@ export const consentUiClasses = {
   iframeFullscreen: 'i-amphtml-consent-ui-iframe-fullscreen',
   iframeActive: 'i-amphtml-consent-ui-iframe-active',
   in: 'i-amphtml-consent-ui-in',
-  loading: 'i-amphtml-consent-loading',
-  fill: 'i-amphtml-consent-fill',
-  placeholder: 'i-amphtml-consent-placeholder',
+  loading: 'i-amphtml-consent-ui-loading',
+  fill: 'i-amphtml-consent-ui-fill',
+  loadingPlaceholder: 'i-amphtml-consent-ui-loading-placeholder',
```

Comment:
Nit: Well, placeholder by default means loading placeholder to me. Can we name it `placeholder: i-amphtml-consent-ui-placeholder` instead : ) A shorter name